libpod/runtime.go

```diff
@@ -42,23 +42,32 @@ const (
 	SQLiteStateStore RuntimeStateStore = iota
 	// BoltDBStateStore is a state backed by a BoltDB database
 	BoltDBStateStore RuntimeStateStore = iota
+)
+
+var (
+	// InstallPrefix is the prefix where podman will be installed.
+	// It can be overridden at build time.
+	installPrefix string = "/usr/local"
```

Comment:
Go Lint does not like the `string` here.
installPrefix = "/usr/local"
is preferred.

Reply:
Fixed, and updated my vim setup to lint in the future.